src/openrct2/ride/RideRatings.cpp

```diff
@@ -2269,3 +2271,15 @@ void RatingTuple::setNull()
 {
     excitement = kRideRatingUndefined;
 }
+
+money64 MazeCalculateCost(money64 constructionCost, const Ride& ride, const CoordsXYZ& loc)
```

Comment:
I realise this is coming out of `ride/gentle/Maze.cpp`, but I'm not sure this is the right place for it either. The only users for this function are in `MazePlaceTrackAction.cpp` and `MazeSetTrackAction.cpp`, correct?

Reply:
That is correct, yes.

Reply:
So what do you think I should do with it?

Reply:
I think it would be better to have maze-specific functions in one Maze.cpp unit file. Looking at this PR, it seems the rest of `ride/gentle/Maze.cpp` was entirely paint related.

Do you think we could move more 'meta' functions into such a `ride/gentle/Maze.cpp` file, were we to reinstate it with just `MazeCalculateCost`, for now?